src/styles/components/loginForm.less

```
@@ -217,17 +217,63 @@
 }
+
+/*
+ * Footprint of the fixed `.loginLanguageSelector` plus a visible gap. The
+ * control is 145px wide at its longest label ("(EN) English", measured at
+ * 1440x900) and sits @spacing-ml from the right edge from xl up. It is
+ * `position: fixed` with a transparent background and z-index 25, so page
+ * content scrolls *underneath* it and it also swallows pointer events over its
+ * own box. The 320px sign-in column never reached that corner; the wide
+ * reading column below would, so it reserves the strip.
+ */
+@public-selector-safe-area: 200px;
```

Stylelint (17.14.1)

[error] Unexpected unknown at-rule "`@public-selector-safe-area`:" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)